src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -1150,6 +1150,13 @@ private static string CleanFileName(string name, NamingConfig namingConfig)
 
             return result.TrimStart(' ', '.').TrimEnd(' ');
         }
+
+        private static string GetFirstCharacter(string title)
+        {
+            var firstCharacter = title[..1].FirstCharToUpper();
+
+            return char.IsLetterOrDigit(firstCharacter, 0) ? firstCharacter : "#";
```

Comment:
`#` feels too similar to digits for me and I'd expect titles that start with `0-9` to be in that folder, not sure which character would be better though.